sdk/storage/azure-storage-blobs/NOTICE.txt

```diff
@@ -1,71 +1,36 @@
-azure-storage-blobs
-
 NOTICES AND INFORMATION
 Do Not Translate or Localize
 
-This software incorporates material from third parties. Microsoft makes certain
-open source code available at https://3rdpartysource.microsoft.com, or you may
-send a check or money order for US $5.00, including the product name, the open
-source component name, and version number, to:
+This software incorporates material from third parties.
+Microsoft makes certain open source code available at https://3rdpartysource.microsoft.com,
+or you may send a check or money order for US $5.00, including the product name,
+the open source component name, platform, and version number, to:
 
 Source Code Compliance Team
 Microsoft Corporation
 One Microsoft Way
 Redmond, WA 98052
 USA
 
-Notwithstanding any other terms, you may reverse engineer this software to the
-extent required to debug changes to any libraries licensed under the GNU Lesser
-General Public License.
-
-------------------------------------------------------------------------------
-
-Azure SDK for C++ uses third-party libraries or other resources that may be
-distributed under licenses different than the Azure SDK for C++ software.
-
-In the event that we accidentally failed to list a required notice, please
-bring it to our attention. Post an issue or email us:
-
-           azcppsdkhelp@microsoft.com
-
-The attached notices are provided for information only.
```

Comment:
The SDKs include this language in all NOTICE files.  Is there a reason you want to remove it?

Reply:
@RickWinter The whole file is generated here.

<img width="497" alt="image" src="https://github.com/Azure/azure-sdk-for-cpp/assets/52266153/981f0b91-ae10-4108-8419-3ac3aece7972">

I didn't make any changes to the generated file (I don't think I should).